PowerShellEditorServices.build.ps1

```diff
@@ -129,6 +129,10 @@ task TestPowerShellApi -If { !$script:IsUnix } {
 task Build {
     exec { & $script:dotnetExe build -c $Configuration .\src\PowerShellEditorServices.Host\PowerShellEditorServices.Host.csproj $script:TargetFrameworksParam }
     exec { & $script:dotnetExe build -c $Configuration .\src\PowerShellEditorServices.VSCode\PowerShellEditorServices.VSCode.csproj $script:TargetFrameworksParam }
+    exec { & $script:dotnetExe publish -c $Configuration .\src\PowerShellEditorServices\PowerShellEditorServices.csproj -f netstandard1.6 }
+    Copy-Item $PSScriptRoot\src\PowerShellEditorServices\bin\$Configuration\netstandard1.6\publish\UnixConsoleEcho.dll -Destination $PSScriptRoot\src\PowerShellEditorServices.Host\bin\$Configuration\netstandard1.6
+    Copy-Item $PSScriptRoot\src\PowerShellEditorServices\bin\$Configuration\netstandard1.6\publish\runtimes\osx-64\native\libdisablekeyecho.dylib -Destination $PSScriptRoot\src\PowerShellEditorServices.Host\bin\$Configuration\netstandard1.6
+    Copy-Item $PSScriptRoot\src\PowerShellEditorServices\bin\$Configuration\netstandard1.6\publish\runtimes\linux-64\native\libdisablekeyecho.so -Destination $PSScriptRoot\src\PowerShellEditorServices.Host\bin\$Configuration\netstandard1.6
```

Comment:
This is not my wheelhouse but are we comfortable that this .so (and the above .dylib) will work on all OS distributions supported by VSCode?

Reply:
@rkeithhill Good question, I initially only tested on Ubuntu.  I just pulled the `microsoft/powershell` docker images for `centos7`, `opensuse42.1`,  and `opensuse13.2` to make sure they at least can load the lib and turn off echo.  But that's not a full test, and I don't know enough about docker and Linux to know if what I did is even an accurate test.  I'm pretty comfortable because of how small the code is, but I definitely can't say with absolute certainly that it will work everywhere.

Reply:
I think you could maybe be a little bit more certain on docker by installing vscode in the image along with some vnc server. Then you should be able to rdp/vnc into the container and mess around with VSCode to verify the fix.

It's possibly a bit lengthly, but if you want to be more certain, it will certainly help.

I'm not actually 100% certain if this is possible though - haven't tried it

Reply:
@tylerl0706 Yeah for sure, though it sounds like about the same time and effort as spinning up a vm for each distro.  Either way that's a bit more involved than I have the bandwidth for, but if someone wants to take that up I'm all for it.

Reply:
I'll make some Dockerfiles one of these days that can grab the distros, install PowerShell, VNC, VSCode, and the extension so we have them for next time.